FrontMen/fm-hours
# refactor: rename users to employees

# Changes

> :warning: **Breaking Changes**: Due to collections renaming in the database, this commit is necessary to run the app again.

## Related issues

Solves #51 

## Added

N/A

## Removed

N/A

## Changed

- Renamed all users references to employees (including DB changes)

## How to test

- Navigate through pages and check if there is an error caused by a missing `user` reference.

## Screenshots

N/A
